docs/geedocs/5.2.4/answer/176738.html

```diff
@@ -245,5 +247,16 @@ <h2 id="example_file_locations">Example of File Locations</h2>
 </tr>
 </tbody>
 </table>
+<h2 id="configure_cache_sizes">Configure cache sizes</h2>
+<p><code>gesystemmanager</code> caches information about the assets and asset versions it is working with.
+The default sizes for these caches should work fine for most builds; however, increasing the cache sizes may speed up very large builds at the cost of increased memory usage.
+The cache sizes are controlled by the file <code>misc.xml</code> which can be found under <code>/gevol/assets/.config/misc.xml</code>
```

Comment:
Style: `which can be found under` -> `which can be found at`, since you're listing the whole path.

Reply:
Thanks. Fixed.